<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminUpdatePetOwner</class>
 <widget class="QWidget" name="adminUpdatePetOwner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="back">
       <property name="minimumSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>â¬…</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Profile Information</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="LE_name">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_password">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_address">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_zipcode">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="LE_bio">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_phoneno">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CB_State">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox {
	text-align: center;
}</string>
         </property>
         <item>
          <property name="text">
           <string>State:</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../build-cuddlySys-Desktop-Debug/map.png</normaloff>../../build-cuddlySys-Desktop-Debug/map.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AK</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AZ</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CO</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HI</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ID</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>IL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>IN</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>IA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>KS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>KY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>LA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ME</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MI</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MN</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MO</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NH</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NJ</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NM</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ND</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>OH</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>OK</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>OR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RI</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TN</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TX</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WI</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WY</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CB_Gender">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <item>
          <property name="text">
           <string>Gender:</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../build-cuddlySys-Desktop-Debug/gender.png</normaloff>../../build-cuddlySys-Desktop-Debug/gender.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Male</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Female</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Other</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Matchmaking Preferences</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QComboBox" name="radius">
         <property name="minimumSize">
          <size>
           <width>179</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Location:</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;5 miles</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;10 miles</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;20 miles</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&gt;20 miles</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Within State</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Out of State</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="species">
         <property name="minimumSize">
          <size>
           <width>179</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Species:</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dog</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cat</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hamster</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fish</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mice</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Birds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Snakes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Other</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_breed">
         <property name="minimumSize">
          <size>
           <width>179</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QComboBox" name="color">
         <property name="minimumSize">
          <size>
           <width>179</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Color:</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Brown</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Black</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>White</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gray</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Other</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="age">
         <property name="minimumSize">
          <size>
           <width>179</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Age:</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Born</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 - 2 years</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2 - 6 years</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6 - 10 years</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10 - 14 years</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14 years and older</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="weight">
         <property name="minimumSize">
          <size>
           <width>179</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Weight:</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;10 lbs</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20 - 50 lbs</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>50 - 100 lbs</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&gt;100 lbs</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="isRescued">
         <property name="maximumSize">
          <size>
           <width>15</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rescue">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Rescued?</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="isDiseased">
         <property name="maximumSize">
          <size>
           <width>15</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="disease">
         <property name="font">
          <font>
           <family>Manjari</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Diseased?</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="PB_update">
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 40px;
    background-color: #9fff80;
    min-width: 80px;
	min-height: 80px;
}</string>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
